src/Symfony/Component/HttpKernel/Kernel.php

```diff
@@ -344,6 +344,30 @@ public function setAnnotatedClassCache(array $annotatedClasses)
     }
 
     /**
+     * Return a list of classes to be cached away when the container is build.
+     *
+     * If a class contains annotations, add it to getAnnotatedClassesToCompile()instead.
```

Comment:
I think its worth mentioning it here, although i would word it like so "Do not add classes that contain annotations, use ... instead"